packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -271,24 +311,24 @@ async def _abort_upload(
         async with session.post(upload_links.links.abort_upload) as resp:
             resp.raise_for_status()
     except ClientError:
-        log.warning("Error while aborting upload", exc_info=True)
+        _logger.warning("Error while aborting upload", exc_info=True)
         if reraise_exceptions:
             raise
-    log.warning("Upload aborted")
+    _logger.warning("Upload aborted")
 
 
-async def upload_file(
+async def upload_path(
     *,
     user_id: UserID,
     store_id: LocationID | None,
     store_name: LocationName | None,
     s3_object: StorageFileID,
-    file_to_upload: Path | UploadableFileObject,
+    path_to_upload: Path | UploadableFileObject,
     io_log_redirect_cb: LogRedirectCB | None,
     client_session: ClientSession | None = None,
     r_clone_settings: RCloneSettings | None = None,
     progress_bar: ProgressBarData | None = None,
-) -> tuple[LocationID, ETag]:
+) -> tuple[LocationID, ETag | None]:

```

Comment:
Suggestion: why  not returning something more informative than `ETag | None`. Something like `FileUploaded | FolderUploaded`.. Both indicate the two possible successful outcomes for uploading a path. Each outcome has the necessary information for that state ie.

```
@dataclass
class FileUploaded:
    store_id: LocationID
    etag: ETag

@dataclass
class FolderUploaded:
    store_id: LocationID

```

Moreover, it give you space to add more attributes in the future for those actions